src/rev_ai/models/job_options.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+"""Job Submit Options Model"""
+
+
+class JobSubmitOptions:
+    def __init__(self,  metadata=None, callback_url=None, skip_diarization=None):
```

Comment:
I think `skip_diarization=False` is better. we're not going to use type annotations, but it shows that it is a boolean from the function sig